cluster/gce/util.sh

```diff
@@ -1254,6 +1254,8 @@ ${CUSTOM_CALICO_NODE_DAEMONSET_YAML//\'/\'\'}
 CUSTOM_TYPHA_DEPLOYMENT_YAML: |
 ${CUSTOM_TYPHA_DEPLOYMENT_YAML//\'/\'\'}
 CONCURRENT_SERVICE_SYNCS: $(yaml-quote "${CONCURRENT_SERVICE_SYNCS:-}")
+ETCD_RUNASUSER: $(yaml-quote "${ETCD_RUNASUSER:-2000}")
```

Comment:
If this is not set then this should be defaulted to 0.

Reply:
There is already an etcd user as 2000, I'm afraid of defaulting it to 0 will end up with running as root again.

Reply:
Where is the etcd user 2000 created?

Reply:
I can't find it anywhere in the code but when I ssh to the master node, I can see the etcd user by `cat /etc/group`.

Reply:
Fixed.